src/Infrastructure.EntityFramework/Repositories/DatabaseContext.cs

```diff
@@ -90,13 +92,17 @@ protected override void OnModelCreating(ModelBuilder builder)
             eUser.Property(c => c.Id).ValueGeneratedNever();
             eOrganizationApiKey.Property(c => c.Id).ValueGeneratedNever();
             eOrganizationConnection.Property(c => c.Id).ValueGeneratedNever();
+            eSecret.Property(c => c.Id).ValueGeneratedNever();
 
             eCollectionCipher.HasKey(cc => new { cc.CollectionId, cc.CipherId });
             eCollectionUser.HasKey(cu => new { cu.CollectionId, cu.OrganizationUserId });
             eCollectionGroup.HasKey(cg => new { cg.CollectionId, cg.GroupId });
             eGrant.HasKey(x => x.Key);
             eGroupUser.HasKey(gu => new { gu.GroupId, gu.OrganizationUserId });
+            eSecret.HasKey(s => s.Id).IsClustered();
 
```

Comment:
```suggestion

            eSecret.HasKey(s => s.Id).IsClustered();
```

Reply:
I wonder if we should investigate using https://docs.microsoft.com/en-us/ef/core/modeling/#grouping-configuration to scale this file.

Reply:
I think using EntityTypeConfiguration files is a good idea and will be simple to add more configurations in the future.

Should now be updated in the latest commit.